README.md

```diff
@@ -305,6 +284,44 @@ to Miri failing to detect cases of undefined behavior in a program.
   `-Zmiri-disable-validation` is set.
```

Comment:
Huh, why is env var isolation affected by validation?

Reply:
Oh lol that should be talking about isolation, not validation.^^